src/main/java/net/minecraftforge/client/event/RenderLivingEvent.java

```diff
@@ -28,35 +28,54 @@
 {
     private final EntityLivingBase entity;
     private final RenderLivingBase<T> renderer;
+    private final float partialRenderTick;
     private final double x;
     private final double y;
     private final double z;
 
+    /*
+        Since adding partialRenderTick causes binary incompatibility,
+        this overload adds a partialRenderTick that simply acts like
+        the game has no values it needs to interpolate.
+     */
+    @Deprecated
     public RenderLivingEvent(EntityLivingBase entity, RenderLivingBase<T> renderer, double x, double y, double z)
     {
+        this(entity, renderer, 1, x, y, z);
```

Comment:
`Minecraft#getRenderPartialTicks()` is probably a better default value than `1`.